src/core/event/event.server.ts

```diff
@@ -53,7 +54,7 @@ export interface ServerEvent extends EventUtilities {
   [GameEventIdentifiers.CardUseEvent]: {
     fromId: PlayerId;
     cardId: CardId;
-    toIds?: PlayerId[];
+    targetGroup?: TargetGroupSet;
```

Comment:
not recommend directly adding a class instance into an event object, need to specify the properties needed from the instance